schema/v1.1.0/dataset_config_models_extended.py

```diff
@@ -268,36 +274,75 @@ async def is_id_ancestor(id_ancestor: str, id: str) -> bool:
         return response.status == 200 and id_ancestor in ancestor_ids
 
 
+@alru_cache
+async def validate_wormbase_id(id: str) -> Tuple[List[str], bool]:
+    """
+    Returns a tuple of the ID names (including original label) and whether or not it is valid.
+    """
+
+    url = f"http://rest.wormbase.org/rest/field/strain/{id}/name"
+
+    logger.debug("Getting ID %s at %s", id, url)
+
+    names = []
+    async with aiohttp.ClientSession() as session, session.get(url) as response:
+        if response.status >= 400:
+            return [], False
+        data = await response.json()
+        if label := data.get("name", {}).get("data", {}).get("label", ""):
+            names.append(label.lower())
```

Comment:
Did we want the names to be case insensitive?

Reply:
Resolved!